### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by spencer-sfdc**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/spencer-sfdc/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (spencer-sfdc) [inline ReadMe.md:1]: I can write something here.

But _can_ I write markdown?

``` js
console.log("yes");
```

Comment (spencer-sfdc) [inline ReadMe.md:1]: !fix

Comment (spencer-sfdc) [inline ReadMe.md:1]: !fix

Comment (spencer-sfdc) [inline ReadMe.md:1]: !fixed

**Pull request merged**